code/index.html

```diff
@@ -4,9 +4,72 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@400;500;600&display=swap" rel="stylesheet">
     <title>Project 4: Pizzabot</title>
   </head>
   <body>
-    <h1>This is your Pizza Bot</h1>
+    <main>
+      <section>
+        <div class="welcome">
+          <h1>Claire & Claudia's pizza ordering service &#127829</h1>
+        </div>
+
+        <!-- Greeting welcoming the customer -->
+        <div class="cards">
+          <h3 id="greeting"></h3>
+          <h3 id="menu" class="menu"><li></li></h3>
+        </div>
+
+        <!-- First form section where user will input username & name of pizza -->
+        <div class="cards">
+          <h3> &#129302 Before we get started with your order please enter your username:</h3> 
+          <form action="#" autocomplete="off" onsubmit="enteredPizzaName(event)" method="POST" id="formOne" class="form-1">
```

Comment:
You don't need the `action` or `method` attribute here at all since you are using the `onsubmit` to control what happens next.

Those attributes are only used when sending data to a remote server via the default form functionality.  

Later, you will also be doing this in Javascript, which means you probably won't use these attributes again for quite some time.

Reply:
Ok perfect. We wrote it in the beginning as we had only this one form, but then we added the second form section. But noted, that we don't need it as we have the last form that does that!